test/lsp-clangd-test.el

```diff
@@ -58,4 +62,4 @@
                   int k);")
```

Comment:
We should also rename this test to something like: `lsp-clangd-join-region`

Reply:
aint this test meant for usage in all the project?

Reply:
Yeah, but it's used only on clang, or you rename it for clangd or move it to a lsp-test more generic or something like this

Reply:
fair

Reply:
done!